Lib/fontbakery/profiles/fontwerk_conditions.py

```diff
@@ -0,0 +1,269 @@
+
+from fontTools.misc.fixedTools import fixedToFloat
+
+general = {
```

Comment:
These should all be uppercase names, e.g. GENERAL and maybe a bit more descriptive, as it's hard to read in the usage context.

Reply:
as in https://github.com/googlefonts/fontbakery/blob/main/Lib/fontbakery/constants.py